docs/infra/vulnerability-management.md

```diff
@@ -0,0 +1,14 @@
+# Vulnerability Management for Docker Images
+This repository contains a GitHub workflow that allows you to scan Docker images for vulnerabilities. The workflow, named `ci-vulnerability-scans` is located in the directory `.github/workflows`. The goal in scanning the image before pushing it to the repository is so that you can catch any vulnerabilities before deploying the image, ECR scanning takes time and the image can still be used even with vulnerabilities found by Inspector. Also, if you use `scratch` as a base image, ECR is unable to scan the image when it is pushed, which is a known issue. Add this to be part of the PR process so that any pushes to the repo will trigger the scan
+
+A way to ensure that there are smaller surface areas for vulnerabilities, follow this method of building images
+- Build base image with required packages, name it something like `build`
+- Configure app build from the image in the previous step, name it something like `app-build`
+- Create a final image from `scratch` named `release` (ie `from scratch as release`), and copy any needed directories from the `app-build` image
+
+By following this method, your deployment image will have the minimum required directories and files
+
+## How to use Workflow
+The workflow will run whenever there is a push to a PR or when merged to main if there are changes in the `app` direcotry. It is scanning in both cases to ensure there is no issues if a PR is approved on a Friday, but isn't merged till Monday - a CVE could have been found in the time between the last run and the merge.
+
+The job in the file, `image-scans`, is the orchestration job that will run the scans in parallel to speed up execution time. If there is a failure in any of the scans, this main job will fail as well
```

Comment:
Looks like the doc still references the job that got removed. Maybe update or remove this paragraph?